README.md

```diff
@@ -1,2 +1,54 @@
-# get-datetime-details-while-tapping-the-.NET-MAUI-Calendar
-This repository contains a sample of get datetime details while tapping the .NET MAUI Calendar.
+# How to get Datetime details while tapping the .NET MAUI Calendar
+
+This repository contains a sample to get the datetime details while tapping using the [Syncfusion .NET MAUI Calendar](https://help.syncfusion.com/maui/calendar/getting-started) control.
+
+Please refer the KB through this link.
+
+## Syncfusion controls
+
+This project used the following Syncfusion control(s):
```

Comment:
This project used the following Syncfusion control(s):
* [SfCalendar](https://www.syncfusion.com/maui-controls/maui-calendar).

Reply:
Seems like the content in the README file and the comment were the same.